Cargo.lock

```diff
@@ -4769,6 +5169,17 @@ version = "0.2.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "e604eb7b43c06650e854be16a2a03155743d3752dd1c943f6829e26b7a36e382"
 
+[[package]]
+name = "twox-hash"
+version = "1.6.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "04f8ab788026715fa63b31960869617cba39117e520eb415b0139543e325ab59"
+dependencies = [
+ "cfg-if 0.1.10",
+ "rand 0.7.3",
+ "static_assertions",
+]
+
```

Comment:
As you mentioned on Slack this is a concerningly large diff to Cargo.lock. I think that means using the `mysql` crate is a nonstarter unless we can get the dependency versions upgraded to avoid the duplicates.

Reply:
I assume your concern here applies no matter where the crate lives, whether under `src` or under `demo/debezium`?

Reply:
Yeah, unfortunately all the dependencies get mixed together in the same Cargo workspace so it doesn't matter where the crate lives. It looks like the async version of this crate, https://crates.io/crates/mysql_async, is far more up-to-date, and hopefully relatively easy to swap in?

Reply:
It looks like mysql_async is much newer and it was fairly easy to swap out. I'll look at the Cargo.lock changes again in the morning, but it appears that there are far fewer this time around! Also, bonus, learned a fair bit about spawning async tasks today 😄